### ISSUE TITLE
Does the mtcnn script do the real align?

### ISSUE THREAD

**Issue by tonybaigang** [davidsandberg/facenet#838]
Hi, Experts
It seems the script align_dataset_mtcnn.py just do face detection, not did align as in  align_dlib.py which use getAffineTransform to align. 
Why the align_dataset_mtcnn.py do not do real align?

**Comment by caocuong0306** [comment]
No. @davidsandberg already stated that his approach is to "just use the face bounding box and let the model generalize over different face poses." https://github.com/davidsandberg/facenet/issues/93. Take a look at https://github.com/davidsandberg/facenet/issues/329 for further discussion.

**Comment by rgsousa88** [comment]
So, can I train a model using a pre-aligned dataset withou using this script? I mean, I have images only from faces and all ones are aligned... Is it possible or recommended?
